packages/mobile/android/app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
```

Comment:
Why is this file needed?

Reply:
good catch, its meant for another pr, which is general android mobile-upload fixes. this is for the date-picker interestingly enough